charts/zookeeper/templates/zookeeper.yaml

```diff
@@ -43,10 +43,18 @@ spec:
     syncLimit: {{ .Values.config.syncLimit }}
     quorumListenOnAllIPs: {{ .Values.config.quorumListenOnAllIPs }}
   {{- end }}
+  {{- if .Values.ehemeral.enabled }}
```

Comment:
this should be `ephemeral`
Also this check should be removed since the value of ephemeral.enabled needs to be set in all cases

Reply:
Done